debian/source/format

```diff
@@ -0,0 +1 @@
+3.0 (quilt)
```

Comment:
This should be `native` instead of `quilt since debian packaging is part of the repository.